src/drivers/ScrollDriver.js

```diff
@@ -26,10 +26,12 @@ import DriverBase from './DriverBase';
  * @param options.nativeScrollEventThrottle Native animated value changes
  *   will be debounced using this value when mirroring them to the JS value.
  *   Used only if `useNativeDriver` is `true`, defaults to 20ms.
+ * @param customOnScroll Function to be executed in tandem with ours.
  */
 export default class ScrollDriver extends DriverBase {
   constructor(
     options = { useNativeDriver: false, nativeScrollEventThrottle: 20 },
```

Comment:
we're not sending ( overriding the default value ) for the nativeScrollEventThrottle from our scrollview, as we're only sending the useNativeDriver prop. I think this is a bug. Please check and include inside our ScrollView if i'm right ( i know it's not related to the bug, but let's sort it out while we're at it )

Reply:
Why would I change this behavior? I'm not sure what the bug is that you're mentioning.

If we set `useNativeDriver` to `true`, but don't set `nativeScrollEventThrottle`, it defaults to `20`. What's the bug?

Reply:
Also you mention naming in the main comment of the CR, but I don't see any comments regarding naming, could you add them if there are any please